internal/service/ipam/model_ipamsvc_ha_group.go

```diff
@@ -0,0 +1,174 @@
+package ipam
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+
+	"github.com/hashicorp/terraform-plugin-framework-timetypes/timetypes"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/ipam"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type IpamsvcHAGroupModel struct {
+	AnycastConfigId types.String      `tfsdk:"anycast_config_id"`
+	Comment         types.String      `tfsdk:"comment"`
+	CreatedAt       timetypes.RFC3339 `tfsdk:"created_at"`
+	Hosts           types.List        `tfsdk:"hosts"`
+	Id              types.String      `tfsdk:"id"`
+	IpSpace         types.String      `tfsdk:"ip_space"`
+	Mode            types.String      `tfsdk:"mode"`
+	Name            types.String      `tfsdk:"name"`
+	Status          types.String      `tfsdk:"status"`
+	Tags            types.Map         `tfsdk:"tags"`
+	UpdatedAt       timetypes.RFC3339 `tfsdk:"updated_at"`
+}
+
+var IpamsvcHAGroupAttrTypes = map[string]attr.Type{
+	"anycast_config_id": types.StringType,
+	"comment":           types.StringType,
+	"created_at":        timetypes.RFC3339Type{},
+	"hosts":             types.ListType{ElemType: types.ObjectType{AttrTypes: IpamsvcHAGroupHostAttrTypes}},
+	"id":                types.StringType,
+	"ip_space":          types.StringType,
+	"mode":              types.StringType,
+	"name":              types.StringType,
+	"status":            types.StringType,
+	"tags":              types.MapType{ElemType: types.StringType},
+	"updated_at":        timetypes.RFC3339Type{},
+}
+
+var IpamsvcHAGroupResourceSchemaAttributes = map[string]schema.Attribute{
+	"anycast_config_id": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "The resource identifier.",
+	},
+	"comment": schema.StringAttribute{
+		Optional: true,
+		Validators: []validator.String{
+			stringvalidator.LengthBetween(1, 1024),
```

Comment:
```suggestion
			stringvalidator.LengthBetween(0, 1024),
```

Description says, may contain 0 chars as well :)

Reply:
I will fix that. Wondering how the acceptance tests passed locally with this change now :)